src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -97,8 +103,26 @@ const PatientProfilePage = ({
     },
   });
   const [editPatient] = useMutation(EDIT_PATIENT);
+  const [deletePatient] = useMutation(DELETE_PATIENT, {
```

Comment:
I am really not sure if this is best practice with updating the cache. I had to implement a cache update because using `const [deletePatient] = useMutation(DELETE_PATIENT)` was not updating the cache by itself (unlike `editPatient` which is curious).

Reply:
this may be related to the fact that the delete mutation isn't returning the ID as of now? cc @wongcmichelle

Reply:
I updated my backend too with Michelle's changes so this should not be an issue anymore

Reply:
Delete not updating the cache immediately is expected as explained here: https://uwblueprint.slack.com/archives/G0149QDHPEV/p1600823248003100